### PR TITLE
AB#91786 IsEnglandHousing delivery CRM connection.

### PR DESCRIPTION
**Pull request by jjankowski87**

### CODE REVIEW

**Review by dawid-pawulski — APPROVED** [review]

**Review by Carq — APPROVED** [review]

File: src/HE.Investments.FrontDoor.Domain/Project/ProjectEntity.cs [hunk 1]
```diff
@@ -16,13 +16,14 @@ public class ProjectEntity : DomainEntity
     public ProjectEntity(
         FrontDoorProjectId id,
         ProjectName name,
+        bool? isEnglandHousingDelivery = null,
```
Comment (Carq) [inline src/HE.Investments.FrontDoor.Domain/Project/ProjectEntity.cs:19]: Do we need this as a parameter? It should be always true

Comment (jjankowski87) [reply src/HE.Investments.FrontDoor.Domain/Project/ProjectEntity.cs:19]: User can create Project, then go back and change answer - and this is saved in CRM

File: src/HE.Investments.FrontDoor.WWW/Controllers/AccountController.cs [hunk 2]
```diff
@@ -0,0 +1,31 @@
+using HE.Investments.Account.Shared.Routing;
+using HE.Investments.Common;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.FeatureManagement;
+
+namespace HE.Investments.FrontDoor.WWW.Controllers;
+
+[Route("account")]
+public class AccountController : Controller
```
Comment (Carq) [inline src/HE.Investments.FrontDoor.WWW/Controllers/AccountController.cs:9]: Add [AuthorizeWithCompletedPRofile]

Comment (jjankowski87) [reply src/HE.Investments.FrontDoor.WWW/Controllers/AccountController.cs:11]: done

**Pull request merged**